quasar-kotlin/src/test/kotlin/co/paralleluniverse/kotlin/fibers/lang/AbstractTest.kt

```diff
@@ -0,0 +1,149 @@
+/*
+ * Quasar: lightweight threads and actors for the JVM.
+ * Copyright (c) 2015-2017, Parallel Universe Software Co. All rights reserved.
+ *
```

Comment:
Is this a new file? If yes, it should be Copyright 2020 R3 Ltd...

Reply:
Yes, I've just looked through Corda code base. I could not find a suitable pattern to use. One for the Guilds maybe.
I've removed this Copyright notice for now.